packages/salesforcedx-vscode-core/src/orgBrowser/nodeTypes.ts

```diff
@@ -10,26 +10,35 @@ export enum NodeType {
   Org,
   MetadataType,
   MetadataCmp,
-  EmptyNode
+  EmptyNode,
+  Folder
 }
 
 export class BrowserNode extends vscode.TreeItem {
   public children: BrowserNode[] = [];
-  constructor(label: string, public readonly type: NodeType) {
+  public readonly fullName: string;
+  constructor(
+    label: string,
+    public readonly type: NodeType,
+    fullName?: string
+  ) {
     super(label);
     this.type = type;
+    this.fullName = fullName || label;
     switch (this.type) {
       case NodeType.Org:
         this.collapsibleState = vscode.TreeItemCollapsibleState.Expanded;
-        this.tooltip = 'Default Org';
         break;
       case NodeType.MetadataCmp:
         this.collapsibleState = vscode.TreeItemCollapsibleState.None;
-        this.tooltip = 'Metadata Component';
+        this.iconPath = vscode.ThemeIcon.File;
```

Comment:
File and folder icons based on the user's icon theme:

<img width="310" alt="Screen Shot 2019-07-09 at 4 42 40 PM" src="https://user-images.githubusercontent.com/10779873/60981974-2f3cb500-a2ec-11e9-8d64-27e60dcf6a94.png">